content-repo/extra-docs/articles/aws-auth-guide.md

```diff
@@ -32,6 +32,8 @@ a _Trust Relationship_ and establishes a trusted relationship between two resour
 More information regarding [Trust Relationships can be found here.](https://docs.aws.amazon.com/directoryservice/latest/admin-guide/edit_trust.html)
 
 ### How XSOAR uses STS to Authenticate
+The following describes the EC2 instance metadata use case. 
```

Comment:
Not correct. STS can be used in both EC2 and access key scenarios. Please delete this line.